met-api/src/met_api/services/comment_service.py

```diff
@@ -73,11 +78,23 @@ def __form_comment(component_id, comment_text, survey_submission: SubmissionSche
             'submission_id': survey_submission.get('id', None)
         }
 
+    @classmethod
+    def extract_components(cls, survey_form: dict):
+        """Extract components from survey form."""
+        components = list(survey_form.get('components', []))
+
+        if survey_form.get('display') == cls.form_display:
```

Comment:
a few comments about logic will be helpful ; or some tests 
comments will be easy i guess..

Reply:
done